pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-http-request.adoc

```diff
@@ -1,55 +1,70 @@
-= Perform an HTTP Call
+= Performing an HTTP Call
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images
 
-To interact with external services, the policy must use HTTP calls. You can perform HTTP calls in the request and response filters and in the function. To perform an HTTP call, inject the `HttpClient` as follows:
+To interact with external services, the policy must use HTTP calls. You can perform HTTP calls in the `[entrypoint]` function, the request filter, or the response filter. To perform an HTTP call, inject the `HttpClient` as follows:
```

Comment:
I believe calls can not be performed in the entrypoint, only during the request and response handlers.

What happens in the entrypoint and on the `on_request` and `on_response` is the injection of the HTTP client, so it can be later used in the code.